.github/workflows/dapr-deploy.yml

```diff
@@ -3,22 +3,24 @@
 # Licensed under the MIT License.
 # ------------------------------------------------------------
 
-name: deploy-dapr-runtime
+name: deploy-longhaul-release
 
 on:
   push:
     branches:
       - master
+  workflow_dispatch:
```

Comment:
This is just to allow for manually triggering this workflow, something that the weekly one has but this doesn't.